.gitignore

```diff
@@ -20,3 +20,7 @@ doc/api/
 *.js.deps
 *.js.map
 .DS_Store
+
+web/assets/firebase_constants.json
+data_tools/serviceAccount.json
```

Comment:
I wouldn't do this - I don't think that a GitIgnore is sufficent protection to handle service credentials. I'd prefer them to be stored in completely outside the Git tree and explicitly passed in as an argument

Reply:
Ahhh that solves all my secret key mess in other places as well. Fixed it